src/libslic3r/GCodeWriter.cpp

```diff
@@ -309,7 +309,7 @@ std::string GCodeWriter::update_progress(unsigned int num, unsigned int tot, boo
 
 std::string GCodeWriter::toolchange_prefix() const
 {
-    return config.manual_filament_change ? ";" + GCodeProcessor::reserved_tag(GCodeProcessor::ETags::Tool_Change) + "T" :
+    return config.manual_filament_change ? ";" + GCodeProcessor::reserved_tag(GCodeProcessor::ETags::MANUAL_TOOL_CHANGE):
```

Comment:
Is there a reason for dropping the "T" here? The T is currently used when processing the tag as it removes the tag then passes the rest to the original process_T method (See GCodeProcessor::process_tags Line 2122). This could be modified, but I think the T also makes it a bit clearer what the number means.

Reply:
I didn't mean to delete the T, must be deleted by accident.